_posts/2018-05-24-mentorship-open-source-hackillinois.markdown

```diff
@@ -0,0 +1,49 @@
+
+HackIllinois was home to lots of smaller, growing open source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn). We love these open source projects because they get great opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi told me that he and his team solved every issue that he had set aside for HackIllinois 2018!
+
+HackIllinois did not limit students to just contributing to open source, though. Many students developed their own open source projects to give back to the community with their own new ideas. One project, [Puppeteer To Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul) even has 69 stars on Github and was mentioned in a talk at GoogleIO! And it was born right here at HackIllinois.
+
+![The hackers from the Puppeteer to Istanbul project present at closing ceremonies](/assets/article_images/mentors-open-source-hackillinois/puppeteer.jpg "puppeteer")
+
+# Sponsors
+
+Sponsors at HackIllinois had a great presence in Open Source as well! Companies had tons of opportunities to get involved with hackers on their projects. Nearly all companies brought mentors that worked with students on contributing and creating open source projects. Even companies that didn't have their own open source projects were able to work with students, because the mentors that came had experience working on large scale projects and were able to contribute alongside students. The one-on-one attention and guidance that these mentors offered was integral to many of the projects that won prizes at HackIllinois.
+
+Companies also had the option to interact with hackers in many other ways too. Some companies sponsored a prize and were able to present it to the project of their choice at the end of the event, and some companies even had personalized lounges for hackers to work in!
+
+<center>&middot;&middot;&middot;</center>
+
+We're incredibly happy that open source was such a success at HackIllinois 2018, and aim to continue the contributions next year at HackIllinois 2019! In the meantime, make sure to check out [our other recent post](https://blog.hackillinois.org/recap/2018/05/19/hackillinois-2018-recap.html) by staff member Justin Yang recapping the entirety of HackIllinois 2018.
+
+Watch out for registration for HackIllinois 2019 on our website at [hackillinois.org](https://hackillinois.org)! And if you're interested in getting involved as a sponsor or mentor, be sure to reach out to us at contact@hackillinois.org.
```

Comment:
overall, great job! I liked the overall open source summary -- I think it could potentially be combined with Rishi's upcoming open source highlights, just had a few grammar/phrasing here and there.

Reply:
Great job Rohin! This article really puts a spotlight to the OS theme we have going. I do definitely think it should be combined with Rishi's (you said it would be on slack - good stuff). I commented on a few things, mostly grammatical, but a couple other things as well. Also, just as a general idea for the article, maybe try to tone it down on the number of exclamation marks that you use. It would just put a more clean professional tone onto the piece. For example, you use exclamatory sentences a lot as transitions where those sentences could just as well be ended with periods. It's a little picky, but I think it could do good for the article. Overall, really great job! Excited to see this one come out.